packages/vite/src/node/server/index.ts

```diff
@@ -224,6 +225,11 @@ export interface ViteDevServer {
    * Print server urls
    */
   printUrls(): void
+  /**
+   * restart the dev server
+   * @param forceOptimize - force the optimizer to ignore the cache and re-bundle
```

Comment:
Maybe we should have the same docs as in `api-javascript`?

Reply:
done